### PR TITLE
Add PV plots/cuts

### PR DESCRIPTION
**Pull request by ilikmeta**

### CODE REVIEW

File: PWGCF/Flow/Tasks/FlowGFWPbPb.cxx [hunk 1]
```diff
@@ -341,6 +359,33 @@ struct FlowGFWPbPb {
...
+      vtxz = collision.posZ();
+      float zRes = TMath::Sqrt(collision.covZZ());
+      if (zRes > 0.25 && collision.numContrib() < 20)
+        vtxz = -999;
+    }
+
+    auto multNTracksPV = collision.multNTracksPV();
+
+    if (centrality >= 70. || centrality < 0)
+      return 0;
```
Comment (victor-gonzalez) [inline PWGCF/Flow/Tasks/FlowGFWPbPb.cxx:373]: Why not returning `False` instead, it will be more readable

File: PWGCF/Flow/Tasks/FlowGFWPbPb.cxx [hunk 2]
```diff
@@ -341,6 +359,33 @@ struct FlowGFWPbPb {
...
+    if (multTrk < fMultCutLow->Eval(centrality))
+      return 0;
+    if (multTrk > fMultCutHigh->Eval(centrality))
+      return 0;
+
+    // V0A T0A 5 sigma cut
+    if (abs(collision.multFV0A() - fT0AV0AMean->Eval(collision.multFT0A())) > 5 * fT0AV0ASigma->Eval(collision.multFT0A()))
+      return 0;
+
     return 1;
```
Comment (victor-gonzalez) [inline PWGCF/Flow/Tasks/FlowGFWPbPb.cxx:389]: and `True` in here
Readability is a much for further code understanding

**Review by victor-gonzalez — APPROVED** [review]
Please, consider my comments for the next iteration

File: PWGCF/Flow/Tasks/FlowGFWPbPb.cxx (hunk 1 above)
Comment (ilikmeta) [reply PWGCF/Flow/Tasks/FlowGFWPbPb.cxx:373]: You mean that to obtain collisions from 0 to 70 % centrality I should return 1 ?

File: PWGCF/Flow/Tasks/FlowGFWPbPb.cxx (hunk 2 above)
Comment (ilikmeta) [reply PWGCF/Flow/Tasks/FlowGFWPbPb.cxx:389]: Here you mean to have return 0 insted of return 1 in line 389?

**Comment by victor-gonzalez** [comment]
I'm talking about readability of the code
Instead of using `0` use `false` and instead of using `1` use `true`

**Comment by ilikmeta** [comment]
Got it! I will so ASAP. Thanks!

You want me to change `o` to `false` anywhre else in the code or only for those you just mentioned?

**Pull request merged**